source/extensions/transport_sockets/tls/context_impl.cc

```diff
@@ -640,12 +644,15 @@ Envoy::Ssl::ClientValidationStatus ContextImpl::verifyCertificate(
 
 void ContextImpl::incCounter(const Stats::StatName name, absl::string_view value,
                              const Stats::StatName fallback) const {
-  Stats::Counter& counter = Stats::Utility::counterFromElements(
-      scope_, {name, stat_name_set_->getBuiltin(value, fallback)});
-  counter.inc();
+  const Stats::StatName value_stat_name = stat_name_set_->getBuiltin(value, fallback);
+  if (value_stat_name == fallback) {
+    ENVOY_LOG_PERIODIC_MISC(error, std::chrono::minutes(1), "Unexpected {} value: {}",
+                            scope_.symbolTable().toString(name), value);
+  }
+  Stats::Utility::counterFromElements(scope_, {name, value_stat_name}).inc();
 
 #ifdef LOG_BUILTIN_STAT_NAMES
-  std::cerr << absl::StrCat("Builtin ", symbol_table.toString(name), ": ", value, "\n")
+  std::cerr << absl::StrCat("Builtin ", scope_.symbolTable().toString(name), ": ", value, "\n")
```

Comment:
I don't think this is needed anymore. Delete?

Reply:
done